app/lib/attachment_batch.rb

```diff
@@ -78,9 +78,17 @@ def remove_files
             logger.debug { "Deleting #{attachment.path(style)}" }
 
             begin
+              retries ||= 0
               attachment.send(:directory).files.new(key: attachment.path(style)).destroy
-            rescue Fog::Storage::OpenStack::NotFound
-              # Ignore failure to delete a file that has already been deleted
+            rescue => e
+              if e.is_a?(Fog::OpenStack::Storage::NotFound)
+                logger.debug { "Will ignore because file is not found #{attachment.path(style)}" }
```

Comment:
I know this is inconsistenly used in this file, but passing a string does suffice here, just as you did in line 89.

```suggestion
                logger.debug "Will ignore because file is not found #{attachment.path(style)}"
```